docs/modules/ROOT/pages/release-notes/release-notes-31-0.adoc

```diff
@@ -0,0 +1,30 @@
+= Release Notes - Payara Platform Enterprise 5.31.0
+
+== Supported APIs and Applications
+
+ * Java EE 8 Applications
+ * Jakarta EE 8
+ * MicroProfile 4.1
+
+=== Notes
+
+=== New Features
+* [FISH-5646] Add Client Certificate Validation API
+
+=== Bug Fixes
+* [FISH-5711] Fix Could Not Load Formatter Class Error in Payara Micro
+* [FISH-5701] Application does not deploy anymore on Payara Server Docker image
+* [FISH-5695] ArrayIndexOutOfBoundsException when the ConfigProperty value ends in a dollar sign \[Community Contribution\]
+* [FISH-5693] Delimiter for configuring multiple KeyStores and TrustStores must use JVM Platform Delimiter
+* [FISH-5691] Do not propagate non-transactional EM to managed tasks
+* [FISH-5690] Remove duplicate postInvoke handler invocation on failure
```

Comment:
Remove Duplicate postInvoke Handler Invocation on Failure

- Note - maybe this is PostInvoke ? not sure if it's a name and has to stay lowercase